js/layers_object.js

```diff
@@ -60,4 +60,10 @@ let layers_ = {
         popup_text: ["Population", "Elderly rates", "Elderly percentage"],
         legend_array: [2, 3, 4, 8, 11]
     },
+    "Testing Per Confirmed Case": {
+        gsheet_keys: ["CASES", "tests_per_confirmed_case"],
+        legend_fn: getColortests,
+        popup_text: ["Population", "Cases", "Number of test per confirmed case"],
```

Comment:
Should it be `test` or `tests` in `Number of test per confirmed case`.

Reply:
Okay I'll look out for that. 
It's supposed to be tests